backend/src/main/java/ca/bc/gov/nrs/vdyp/backend/v1/utils/FileHelper.java

```diff
@@ -0,0 +1,36 @@
+package ca.bc.gov.nrs.vdyp.backend.v1.utils;
```

Comment:
No accounting for null inputs in these, but I assume that's a pretty low likelyhood